### ISSUE TITLE
Need to know whether whitelist has been added

### ISSUE THREAD

**Issue by dkent600** [PrimeDAO/prime-launch-dapp#246]

**Comment by dkent600** [comment]
@renc-curvelabs Can you please triage?

**Comment by renc-curvelabs** [comment]
_The Seed contract does not allow us to know whether anyone has been whitelisted, like, to confirm that the whitelist has been added._ 

Can't this be confirmed from the registration wizard whether a user have chosen to upload a whitelist or not? 

_I think it might also be fair that we should prevent a seed from being usable until the whitelist has been added._

I am not sure what you mean by this as there will be projects that will not have a whitelist.

**Comment by dkent600** [comment]
> Can't this be confirmed from the registration wizard whether a user have chosen to upload a whitelist or not?

No. Two different things: specifying there is supposed to be a whitelist, and submitting a whitelist (or just whitelisting someone).  Recall that the latter is something the admin must do if a whitelist is required.

> I am not sure what you mean by this as there will be projects that will not have a whitelist.

This question is only relevant to Seeds that have specifying they want a whitelist.

**Comment by dkent600** [comment]
Awaiting change to Seed contract to enable us to work this.  @codebuster22 

Will be a boolean in the Seed contract that indicates whether the batchWhitelist has ever been invoked.

We use that to enable access to the seedDashboard (as part of computing the UNINITIALIZED state)

And we use it in the seed admin dashboard:  When false we show the button to add the whitelist.  When true we show the buttons to add/remove individual accounts to the whitelist.

**Comment by codebuster22** [comment]
Here - https://github.com/PrimeDAO/contracts-v2/pull/175
